libobs/obs-nix-wayland.c

```diff
@@ -0,0 +1,90 @@
+/******************************************************************************
+    Copyright (C) 2019 by Jason Francis <cycl0ps@tuta.io>
+
+    This program is free software: you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation, either version 2 of the License, or
+    (at your option) any later version.
+
+    This program is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License
+    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+******************************************************************************/
+
+#include "obs-internal.h"
+#include "obs-nix-wayland.h"
+
+#include <wayland-client.h>
+
+void obs_nix_wayland_log_info(void)
```

Comment:
???

Reply:
What's wrong with it?

Reply:
It doesnt really print anything helpful. If the platform display is null here it just means it was not set and does not imply that it cannot connect (or even that a connection has been attempted)

printing if a platform display was provided and marking this `TODO: query some information about the wayland server if possible` would probably better.

Reply:
I'm not sure about adding more information... I couldn't find any QT API that returns the underlying `wl_compositor` and doing it manually would be probably not worth all the code.

Reply:
im not sure there is anything worth querying at this point. So thats why i recommend a todo.

Reply:
Added the TODO